src/cli/serve/integration_tests/reload_logging_config.test.js

```diff
@@ -60,16 +60,20 @@ describe('Server logging configuration', function () {
     });
   } else {
     it('should be reloadable via SIGHUP process signaling', function (done) {
-      expect.assertions(6);
+      expect.assertions(1);
```

Comment:
What's the need for this line at all?

Reply:
Oh, because the assertion is called within async code, the callback to `child.on()`

Reply:
But we say `done()`, so maybe it's not strictly necessary to explicitly expect a number of assertions.

Reply:
It's not strictly necessary anymore, I just feel it makes it obvious what's expected in the test, given it's async (so it's more to "guide the reader")